client/src/modules/Profile/Styles/App.css

```diff
@@ -29,6 +51,28 @@
   .scale-logo-homepage {
     margin: 120px 0px 0px 30px;
   }
+
+  .sign-in-button {
+    position: absolute;
+    right: 30px;
+    top: 40px;
+
+    width: 100px;
+    height: 39px;
+
+    background: #ffffff;
+    border-radius: 10px;
+
+    font-family: Source Sans Pro;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 23px;
+    text-align: center;
+    letter-spacing: -0.22px;
+
+    color: #000000;
+  }
```

Comment:
For changing the CSS with media tags, you only need to add the line that is being changed. In this case, you only need to include `width: 100px`, but it's a bit strange to go from percentages to pixels, so could this be changed to a percentage as well? This makes the file shorter and more consistent especially since `App.css` is quite 😢